src/Sylius/Bundle/AdminBundle/TwigComponent/Country/CountryType.php

```diff
@@ -22,7 +22,7 @@
 use Symfony\UX\LiveComponent\DefaultActionTrait;
 use Symfony\UX\LiveComponent\LiveCollectionTrait;
 
-#[AsLiveComponent(name: 'SyliusAdmin.Country.CountryType', template: '@SyliusAdmin/Country/_form.html.twig')]
+#[AsLiveComponent(name: 'SyliusAdmin.Country.CountryType', template: '@SyliusAdmin/Country/_form.html.twig', method: 'post')]
```

Comment:
From what I can see, it's a default value, but I'm fine with explicitly defining it

Reply:
I forgot to remove this.

Reply:
I would also be for always stating the method used for clarity.

Reply:
We will standardize when we turn off autoconfigure.